app.js

```diff
@@ -0,0 +1,67 @@
+const bookSection = document.getElementById('book-section');
+
+function add(e) {
+  const name = document.getElementById('input-name').value;
+  const author = document.getElementById('input-author').value;
+
+  const book = {
+    name,
+    author,
+  };
+
+  if (localStorage.getItem('books') === null) {
+    const books = [];
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  } else {
+    const books = JSON.parse(localStorage.getItem('books'));
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+  document.getElementById('form').reset();
+  e.preventDefault();
+  window.location.reload();
+}
+
+document.getElementById('form').addEventListener('submit', add);
+
+function remove(name) {
+  const books = JSON.parse(localStorage.getItem('books'));
+
+  for (let i = 0; i < books.length; i += 1) {
+    if (name === books[i].name) {
+      books.splice(i, 1);
+    }
+  }
+
+  localStorage.setItem('books', JSON.stringify(books));
+  window.location.reload();
+}
+
+function loadScreen() {
+  const books = JSON.parse(localStorage.getItem('books'));
+  for (let i = 0; i < books.length; i += 1) {
+    const { name } = books[i].name;
+    const { author } = books[i].author;
```

Comment:
- [ ] At the moment, the `remove` and the `loadScreen` functions are not working as they should. I noticed the problem was how the name and `author` were restructured. You can do it this way instead `const { name } = books[i]` and `const { author } = books[i]`.
- [ ] Also I will advise you remove by `id` instead of `name` as two books might have the same `name`